src/wallet/wallet_rpc_server.cpp

```diff
@@ -588,17 +588,17 @@ namespace tools
   {
     if (m_wallet)
     {
-      MDEBUG(tools::wallet_rpc_server::tr("Closing wallet..."));
+      LOG_PRINT_L0(tools::wallet_rpc_server::tr("Closing wallet..."));
       stop_long_poll_thread();
       if (save_current)
       {
-        MDEBUG(tools::wallet_rpc_server::tr("Saving wallet..."));
+        LOG_PRINT_L0(tools::wallet_rpc_server::tr("Saving wallet..."));
         m_wallet->store();
-        MINFO(tools::wallet_rpc_server::tr("Wallet saved"));
+        LOG_PRINT_L0(tools::wallet_rpc_server::tr("Successfully saved"));
       }
       m_wallet->deinit();
       m_wallet.reset();
-      MINFO(tools::wallet_rpc_server::tr("Wallet closed"));
+      LOG_PRINT_L0(tools::wallet_rpc_server::tr("Successfully closed"));
```

Comment:
I'm not sure if this log increase was intentional, but if so can we use `MWARNING` insted of `LOG_PRINT_L0` (the latter is simply a #define alias for the former)?  The numeric ones are far less descriptive and IMO we should be trying to phase them out entirely.

(Actually they probably shouldn't be warning-level as it feels a little weird to see "WARNING: Successfully saved" in the logs).

Reply:
Yeah this is a revert commit that my git made and not intentional. Will remove